pkg/mount/dag_steps.go

```diff
@@ -471,3 +469,37 @@ func (s *State) LoadKernelModules(g *herd.Graph) error {
 		}),
 	)
 }
+
+// WaitForSysrootDagStep waits for the s.Rootdir and s.Rootdir/system paths to be there
+// Useful for livecd/netboot as we want to run steps after s.Rootdir is ready but we don't mount it ourselves.
+func (s *State) WaitForSysrootDagStep(g *herd.Graph) error {
+	return g.Add(cnst.OpWaitForSysroot,
+		herd.WithCallback(func(ctx context.Context) error {
+			cc := time.After(60 * time.Second)
```

Comment:
nit: we could set the timer as a arg on `WaitForSysrootDagStep`, but that's a minor, feel free to ignore